src/App/App.tsx

```diff
@@ -1,9 +1,10 @@
 import * as React from 'react';
+import {BrowserRouter as Router, Switch, Route} from 'react-router-dom';
```

Comment:
I suggest to refuse renaming imports. It makes it more difficult to read and comprehend the code.

Reply:
fixed